index.js

```diff
@@ -54,6 +54,11 @@ function handleArgs(cmd, args, opts) {
 		opts.env = npmRunPath.env(Object.assign({}, opts, {cwd: opts.localDir}));
 	}
 
+	if (opts.detached) {
+		// https://github.com/sindresorhus/execa/issues/115
```

Comment:
Let's just do the issue number `#115`. We highlight it with Refined GitHub, so we don't really need the noise of the full URL.